vnext/local-cli/generator-windows/templates/cpp/proj/MyApp.vcxproj

```diff
@@ -67,6 +67,7 @@
     <WholeProgramOptimization>true</WholeProgramOptimization>
     <LinkIncremental>false</LinkIncremental>
   </PropertyGroup>
+  <%=XamlProps%>
```

Comment:
So the expectation is that people choose WinUI at project creation?

Reply:
@jonthysell - Mixing winui 2.x and 3.0 isn't really supported (read: tested) so using RNW with winui2.x and then want to use winui 3 in your app or vice versa, you end up with undefined behavior AFAICT. Plus you don't want to load both WUX and MUX dlls in your process.
I could have written this in a way that exposed WinUI 3 as a build configuration option but this seemed less prone for folks to get into a bad mix situation, it's either 2.x or 3.